linera-core/src/unit_tests/wasm_worker_tests.rs

```diff
@@ -296,14 +296,18 @@ where
         )
         .await?;
     creator_state.system.timestamp.set(Timestamp::from(3));
+    creator_state
+        .system
+        .used_blobs
+        .insert(&application_description_blob_id)?;
     let run_block_proposal = ConfirmedBlock::new(
         BlockExecutionOutcome {
             messages: vec![Vec::new()],
             previous_message_blocks: BTreeMap::new(),
             events: vec![Vec::new()],
             blobs: vec![Vec::new()],
             state_hash: creator_state.crypto_hash().await?,
-            oracle_responses: vec![vec![OracleResponse::Blob(application_description_blob_id)]],
+            oracle_responses: vec![vec![]],
```

Comment:
I don't understand what has changed in the code that this blob was moved from `oracle_responses` to `used_blobs` collection. The latter is not verified by the protocol?

Reply:
Since we already add it to `used_blobs` when creating the application, no further oracle responses are generated when it is used later.

(At first I thought this would cause a problem, but `requires_blob` already takes the created blobs into account, too, so no oracle response should be necessary here.)